analyzer/pkg/rpm/rpm_test.go

```diff
@@ -16,15 +16,34 @@ func TestParseRpmInfo(t *testing.T) {
 		path string
 		pkgs []types.Package
 	}{
-		"Valid": {
-			path: "./testdata/valid",
-			// cp ./testdata/valid /path/to/testdir/Packages
+		"Valid CentOS": {
+			path: "./testdata/valid_centos",
+			// cp ./testdata/valid_centos /path/to/testdir/Packages
 			// rpm --dbpath /path/to/testdir -qa --qf "{Name: \"%{NAME}\", Epoch: %{EPOCHNUM}, Version: \"%{VERSION}\", Release: \"%{RELEASE}\", Arch: \"%{ARCH}\"\},\n"
 			pkgs: []types.Package{
 				{Name: "centos-release", Epoch: 0, Version: "7", Release: "1.1503.el7.centos.2.8", Arch: "x86_64", SrcName: "centos-release", SrcEpoch: 0, SrcVersion: "7", SrcRelease: "1.1503.el7.centos.2.8"},
 				{Name: "filesystem", Epoch: 0, Version: "3.2", Release: "18.el7", Arch: "x86_64", SrcName: "filesystem", SrcEpoch: 0, SrcVersion: "3.2", SrcRelease: "18.el7"},
 			},
 		},
+		"Valid Rocky Linux": {
+			path: "./testdata/valid_rocky",
+			// cp ./testdata/valid_rocky /path/to/testdir/Packages
+			// rpm --dbpath /path/to/testdir -qa --qf "{Name: \"%{NAME}\", Epoch: %{EPOCHNUM}, Version: \"%{VERSION}\", Release: \"%{RELEASE}\", Arch: \"%{ARCH}\"\},\n"
+			pkgs: []types.Package{
+				{Name: "python3-libs", Epoch: 0, Version: "3.6.8", Release: "37.el8.rocky", Arch: "x86_64", SrcName: "python3", SrcEpoch: 0, SrcVersion: "3.6.8", SrcRelease: "37.el8.rocky"},
+				{Name: "rocky-release", Epoch: 0, Version: "8.4", Release: "26.el8", Arch: "noarch", SrcName: "rocky-release", SrcEpoch: 0, SrcVersion: "8.4", SrcRelease: "26.el8"},
+			},
+		},
+		"Valid Almainux": {
+			path: "./testdata/valid_alma",
+			// cp ./testdata/valid_alma /path/to/testdir/Packages
+			// rpm --dbpath /path/to/testdir -qa --qf "{Name: \"%{NAME}\", Epoch: %{EPOCHNUM}, Version: \"%{VERSION}\", Release: \"%{RELEASE}\", Arch: \"%{ARCH}\"\},\n"
+			pkgs: []types.Package{
+				{Name: "libdnf", Epoch: 0, Version: "0.55.0", Release: "7.el8.alma", Arch: "x86_64", SrcName: "libdnf", SrcEpoch: 0, SrcVersion: "0.55.0", SrcRelease: "7.el8.alma"},
+				{Name: "python3-libdnf", Epoch: 0, Version: "0.55.0", Release: "7.el8.alma", Arch: "x86_64", SrcName: "libdnf", SrcEpoch: 0, SrcVersion: "0.55.0", SrcRelease: "7.el8.alma"},
+				{Name: "almalinux-release", Epoch: 0, Version: "8.4", Release: "2.el8", Arch: "x86_64", SrcName: "almalinux-release", SrcEpoch: 0, SrcVersion: "8.4", SrcRelease: "2.el8"},
+			},
+		},
```

Comment:
Do AlmaLinux and Rochy Linux have any differences from RHEL/CentOS? If no, we don't have to add tests.

Reply:
There is almost no difference between AlmaLinux, Rocky Linux and RHEL/CentOS based on the information obtained from rpm. The only difference is that `alma` and `rocky` are included in Release, as in CentOS.

Reply:
In this test case, it shows that each Release contains `.alma` and `.rocky`, and that it can parse them into a Release.